modules/file-system/parallelstore/scripts/mount-daos.sh

```diff
@@ -73,12 +74,34 @@ for i in {1..10}; do
 	# shellcheck disable=SC2086
 	dfuse -m "$local_mount" --pool default-pool --container default-container --multi-user $mount_options && break
 
-	echo "dfuse failed, retrying in 1 seconds (attempt $i/5)..."
+	echo "dfuse failed, retrying in 1 seconds (attempt $i/10)..."
 	sleep 1
 done
 
 if ! mountpoint -q "$local_mount"; then
 	exit 1
 fi
 
+# Store the mounting logic in a variable
+mount_command='for i in {1..10}; do /bin/dfuse -m '$local_mount' --pool default-pool --container default-container --multi-user '$mount_options' --foreground && break; echo \"dfuse, failed, retrying in 1 second (attempt '$i'/10)\"; sleep 1; done'
+
+# --- Begin: Add systemd service creation ---
+cat >/usr/lib/systemd/system/mount_parallelstore.service <<EOF
```

Comment:
Does this name should contain `$local_mount` to enable mounting multiple parallel stores?

Reply:
ATM, mounting multiple parallelstore instance to the same cluster is not supported.